File: src/main/scala/com/cokeframework/elasticsearch/annotation/FieldType.scala
package com.cokeframework.elasticsearch.annotation

object FieldType extends Enumeration {
  type FieldType = Value
  val Text = Value("Text")
  val Integer = Value("Integer")
  val Long = Value("Long")
  val Double = Value("Double")
  val Date = Value("Date")
  val Boolean = Value("Boolean")
  val Keyword = Value("Keyword")
}

File: src/main/scala/com/kanke/ml/annotation/FieldType.java
package com.kanke.ml.annotation;

public enum FieldType {

    Text("Text"),
    Long("Long"),
    Auto("Auto"),
    Keyword("Keyword"),
    Double("Double"),
    Date("Date"),
    Boolean("Boolean"),
    Integer("Integer");
    private final String fieldType;
    private FieldType(String fieldType) {
        this.fieldType = fieldType;
    }
}
